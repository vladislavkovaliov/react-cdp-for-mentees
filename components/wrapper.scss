@import '../constants/_variables.scss';

.wrapper {
  max-width: 1178px;
  width: 100%;
  margin: 0 auto;
  flex: 1 0 auto;
}

