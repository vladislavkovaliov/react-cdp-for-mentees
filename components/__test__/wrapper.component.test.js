import React from 'react';
import { Wrapper } from '../wrapper.component';
import { shallow } from 'enzyme';

describe('<Wrapper />', () => {
  describe('Shapshot', () => {
    test('should render <Wrapper /> without children', () => {
      const wrapper = shallow(
        <Wrapper />
      );

      expect(wrapper).toMatchSnapshot();
    });

    test('should render <Wrapper /> with children', () => {
      const wrapper = shallow(
        <Wrapper>
          <div>mocks children</div>
        </Wrapper>
      );

      expect(wrapper).toMatchSnapshot();
    });
  });
});