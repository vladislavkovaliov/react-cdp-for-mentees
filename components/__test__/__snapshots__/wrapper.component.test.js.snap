// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> Shapshot should render <Wrapper /> with children 1`] = `
<div
  className="wrapper"
>
  <div>
    mocks children
  </div>
</div>
`;

exports[`<Wrapper /> Shapshot should render <Wrapper /> without children 1`] = `
<div
  className="wrapper"
/>
`;
